.github/workflows/deploy-demo.yaml

```diff
@@ -0,0 +1,37 @@
+name: "Deploy Demo"
+on:
+  workflow_dispatch: {}
+
+env:
+  REGISTRY: ghcr.io
+
+jobs:
+  deployment:
+    runs-on: 'ubuntu-latest'
+    container: dtzar/helm-kubectl:3.9.4
+    env:
+      DEPLOY_IMAGE: ghcr.io/${{ github.repository }}
+      WORKER_IMAGE: ghcr.io/${{ github.repository }}/worker
+      HELM_EXPERIMENTAL_OCI: 1
+      HELM_RELEASE_NAME: gbh-demo
+      KUBE_NAMESPACE: gbh-demo
+      HELM_EXTRA_ARGS: >
+        --values ops/demo-deploy.yaml
+      KUBECONFIG_FILE: ${{ secrets.KUBECONFIG_FILE_DEMO }}
+      KUBECONFIG: ./kubeconfig.yml
+      SOLR_PASSWORD: ${{ secrets.SOLR_PASSWORD_DEMO }}
+      MYSQL_PASSWORD: ${{ secrets.MYSQL_PASSWORD_DEMO }}
+      MYSQL_ROOT_PASSWORD: ${{ secrets.MYSQL_ROOT_PASSWORD }}
+    steps:
+      - name: Set env
+        run: >-
+          echo "TAG=${GITHUB_SHA::8}" >> $GITHUB_ENV;
+          echo "DEPLOY_TAG=${GITHUB_SHA::8}" >> $GITHUB_ENV;
+      - name: Checkout code
+        uses: actions/checkout@v3
+      - name: Do deploy
+        run: >-
+          echo $KUBECONFIG_FILE | base64 -d > $KUBECONFIG;
+          DOLLAR=$ envsubst < ops/demo-deploy.tmpl.yaml > ops/demo-deploy.yaml;
+          cat ops/demo-deploy.yaml;
+          ./bin/helm_deploy gbh-demo gbh-demo
```

Comment:
👍

Reply:
this was one of those validation errors that Tim keeps talking about. A lot of the example data sets would fail on this error, but the records seem to be otherwise valid. I thought that the goal was to let the ams1 stuff in close to as is at this point. This may need requirement discussion?